src/App.tsx

```diff
@@ -186,37 +187,31 @@ export default function App() {
             .join('')};
         </div> */}
       {choices?.length > 0 && (
-        <div
-          ref={scrollRef}
+        <SimpleBar
+          scrollableNodeProps={{ ref: scrollRef }}
```

Comment:
I added [simplebar](https://www.npmjs.com/package/simplebar-react) because scroll view is such a crucial part of the GUI and native one looks rather ugly. it auto hides, too.

<img src="https://user-images.githubusercontent.com/25487857/107012352-34e4d000-6799-11eb-99de-0487176e0f5a.png" width="350">